### ISSUE TITLE
Kernel version parsing fails with certain version suffixes

### ISSUE THREAD

**Issue by aoldershaw** [cloudfoundry/guardian#232]
## Description

Guardian fails to start up when the kernel version has certain suffixes (e.g. `5.4.89+`)

## Logging and/or test output

```
{"timestamp":"2021-05-26T14:57:45.847744323Z","level":"error","source":"guardian","message":"guardian.failed-to-wire-containerizer","data":{"error":"Malformed version: 5.4.89+"}}
Malformed version: 5.4.89+
Malformed version: 5.4.89+
```

## Steps to reproduce

Outline the steps to test or reproduce the PR here. Please also provide the
following information if applicable:

* Guardian release version: `f2e380e2872d8a2c1aaffa95239a54492e290fcf` (output of `gdn version -v`) / `1.19.26`
* Linux kernel version `5.4.89+`
* Concourse version `7.3.0`

**Comment by analytically** [comment]
Having the same issue after upgrading to 7.3.0: `{"timestamp":"2021-05-26T20:09:53.948392112Z","level":"error","source":"guardian","message":"guardian.failed-to-wire-containerizer","data":{"error":"Malformed version: 5.10.35-31.135.amzn2.x86_64"}}`

**Comment by danail-branekov** [comment]
Hey @aoldershaw,

Thanks for raising the issue and submitting the PR. It has been released with garden-runc-release [v1.19.28](https://github.com/cloudfoundry/garden-runc-release/releases/tag/v1.19.28)

**Comment by aoldershaw** [comment]
Awesome, thanks for getting that out so quick 😄
